src/index.html

```diff
@@ -2,16 +2,267 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
     <script type="text/javascript" src="./main.js"></script>
+    <header class="header"></header>
+
+    <main class="main">
+      <div class="content">
+        <form
+        action="http://localhost:8080/api" method="post"
+        name="taskForm"
+        >
```

Comment:
```suggestion
        <form
          action="http://localhost:8080/api" method="post"
          name="taskForm"
        >
```
check code style
<img width="920" alt="image" src="https://user-images.githubusercontent.com/102411130/219582139-04d93cb4-f3ee-41f3-b095-6c2ee653e26f.png">